build/jobs/run-tests.yml

```diff
@@ -51,8 +44,7 @@ jobs:
       'CosmosDb:Key': $(CosmosDb--Key)
       'SqlServer:ConnectionString': $(SqlServer--ConnectionString)
 
-- job: 'e2eTests'
-  condition: eq('${{ parameters.runE2E }}', true)
+- job: 'cosmosE2eTests'
```

Comment:
Is it possible to align the naming with the AKS version at all?
`R4 SqlServer AKS Tests e2eTests`

Reply:
The AKS tests are split at the stage level, while these are split at the job level. I can change the names, but they are already under a stage that references the version being run.